src/nora/components/Tag/Tag.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '../Attributes.css';
```

Comment:
@roblevintennis is this import necessary? I couldn't get `--Radius-3` to work without this explicit import, but didn't have to do this with the colors

Reply:
Yeah, I actually think it IS required so I think you're good the way you have it. Only way it wouldn't be is if we unconditionally were loading those definitions which we are not. I prefer module imports where possible anyway, so I like the pattern you're using here 👍